docs/reference/cluster/update-settings.asciidoc

```diff
@@ -45,6 +45,11 @@ Cluster wide settings can be returned using:
 curl -XGET localhost:9200/_cluster/settings
 --------------------------------------------------
 
+If the same setting is made both in the elasticsearch.yml configuration
+file and as a persistent cluster setting, the persistent cluster file
+setting will take presedence.  For this reason it is preferrable to 
```

Comment:
precedence - which one will be used.  "take precedence" => "has priority" ?

On Thu, Nov 5, 2015 at 2:44 PM Luca Cavanna <notifications@github.com>
wrote:

> In docs/reference/cluster/update-settings.asciidoc
> <https://github.com/elastic/elasticsearch/pull/14559#discussion_r44011880>
> :
>
> > @@ -45,6 +45,11 @@ Cluster wide settings can be returned using:
> >  curl -XGET localhost:9200/_cluster/settings
> >  --------------------------------------------------
> >
> > +If the same setting is made both in the elasticsearch.yml configuration
> > +file and as a persistent cluster setting, the persistent cluster file
> > +setting will take presedence.  For this reason it is preferrable to
>
> precedence?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/elasticsearch/pull/14559/files#r44011880>.
>